### ISSUE TITLE
Pigtail Mesh & Explaination

### ISSUE THREAD

**Issue by hippyau** [mvrdevelopment/spec#238]
**Is your feature request related to a problem? Please describe.**

Table 32. *Model Attributes*

PrimitiveType  “Pigtail” is referred to in the specification, but no explanation of what it is nor a mesh is provided.

**Describe the solution you'd like**
A clear and concise description of what a pigtail is, along with a mesh provided for it.

**Describe alternatives you've considered**
I've looked at another pigtail mesh, which appears to be a cube?  I understand it is the back of a fixture where the IO connects, but I could be wrong, as it's not described.

**Comment by Verschwiegener** [comment]
The way I understand it, the Pigtail is just a Geometry for all IO Connectors. Basically the Panel with all Connectors of a Fixture. And because the Connectors aren't modeled the Pigtail also need no Model or just a cube.

**Comment by hippyau** [comment]
Cheers, yep thanks.  Just that `Pigtail` is defined as a PrimitiveType with no explanation that the primitive shape is a cube or whatever. 

```
| PrimitiveType      | Enum | Type of 3D model; The currently defined values are: “Undefined”, “Cube”, “Cylinder”, “Sphere”, “Base”, “Yoke”, “Head”, “Scanner”, “Conventional”, “Pigtail”, "Base1_1", "Scanner1_1", "Conventional1_1"; 
```


Revision history shows:

```
## Version 0.87
[......]
 - Pigtail position should not be specified in 2D or 3D files anymore.
    Instead of it should be created a general geometry and linked to a
    model with primitive type “Pigtail”;
```

**Comment by Verschwiegener** [comment]
Yeah i think the  Mesh shape is irrelevant the Pigtail is more just the Name for a Geometry of IO Components
